### PR TITLE
feat(x/precisebank): Add SpendableCoin method

### PR DESCRIPTION
**Pull request by drklee3**
## Description
- Change `GetBalance()` to return full balances instead of spendable
- Add `SpendableCoin()` method with support of akava

### CODE REVIEW

File: x/precisebank/keeper/view_integration_test.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+			),
+		},
+		{
+			"non-extended denom - ukava returns ukava",
+			types.IntegerCoinDenom,
+			sdk.NewCoins(sdk.NewCoin(types.IntegerCoinDenom, sdk.NewInt(1000))),
+			sdk.ZeroInt(),
+			sdk.NewCoins(sdk.NewCoin(types.IntegerCoinDenom, sdk.NewInt(10))),
+			sdk.NewCoin(types.IntegerCoinDenom, sdk.NewInt(990)),
+		},
```
Comment (pirtleshell) [inline x/precisebank/keeper/view_integration_test.go:72]: can we add a similar test case, but with non-zero fractional balance?
i could see it preventing a future regression

**Review by pirtleshell — APPROVED** [review]
LGTM 👍
